server/ai_http.go

```diff
@@ -46,6 +46,7 @@ func startAIServer(lp lphttp) error {
 	lp.transRPC.Handle("/audio-to-text", oapiReqValidator(lp.AudioToText()))
 	lp.transRPC.Handle("/llm", oapiReqValidator(lp.LLM()))
 	lp.transRPC.Handle("/segment-anything-2", oapiReqValidator(lp.SegmentAnything2()))
+	lp.transRPC.Handle("/realtime-to-realtime", oapiReqValidator(lp.StartRealtimeToRealtime()))
```

Comment:
Will this endpoint be used by MediaMTX to send segments? And then receive synchronously segments?